load/scan.go

```diff
@@ -99,28 +102,35 @@ func scanWithIndexer(channels []*duplexChannel, batchSize uint, limit uint64, br
 	}
 
 	// Batches that are ready to be set when space on a channel opens
-	unsent := make([][]Batch, numChannels)
-	for i := range unsent {
-		unsent[i] = []Batch{}
+	outstandingBatches := make([][]Batch, numChannels)
+	for i := range outstandingBatches {
+		outstandingBatches[i] = []Batch{}
 	}
 
 	// We use Select via reflection to either select an acknowledged channel so
 	// that we can potentially send another batch, or if none are ready to continue
 	// on scanning. However, when we reach a limit of outstanding batches, we also
 	// want to block until one worker is done, so as not to starve the workers.
-	// Using an array with Select via reflection gives us this flexibility (i.e., we can
-	// either pass the whole array of cases, or the array less the last item).
-	cases := make([]reflect.SelectCase, numChannels+1)
+	// Using an array with Select via reflection gives us this flexibility (i.e.,
+	// we can either pass the whole array of cases, or the array less the last item).
+	cases := make([]reflect.SelectCase, numChannels + 1)
```

Comment:
pretty sure this is not `gofmt`. While I don't like that it is smushed together, this is just going to revert back to that once `gofmt` is run again.

Reply:
Ok, agreed. `gofmt` did indeed smushed this piece as `numChannels+1`. let it be so
Summary: updated